chunky/src/java/se/llbit/chunky/renderer/scene/SampleBuffer.java

```diff
@@ -0,0 +1,125 @@
+/* Copyright (c) 2021 Chunky contributors
+ *
+ * This file is part of Chunky.
+ *
+ * Chunky is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Chunky is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with Chunky.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package se.llbit.chunky.renderer.scene;
+
+import se.llbit.math.ColorUtil;
+
+public class SampleBuffer {
+  public final int width;   // = rowSize/3, used for alpha
```

Comment:
"used for alpha"?

Reply:
Alpha buffer has only 1 value for pixel, where as "rowSize" is 3x as much since R G and B samples in the main array. Fixed to be clearer in next commit.